src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1530,8 +1530,8 @@ Additional information: {1}</value>
   <data name="Comments" xml:space="preserve">
     <value>Comments</value>
   </data>
-  <data name="Inline_Hints" xml:space="preserve">
-    <value>Inline Hints</value>
+  <data name="Inlay_Hints" xml:space="preserve">
+    <value>Inlay Hints</value>
   </data>
```

Comment:
There should be a set of strings somewhere used for text search; should we still add the old name so people can find that way?

Reply:
Good idea. I'm not sure of what the property is though (and I cannot seem to find docs explaining it either).

Reply:
I think this is what you need
https://github.com/dotnet/roslyn/blob/580129a61f2cb88be53b7b225c00b8bd9af1d497/src/VisualStudio/CSharp/Impl/VSPackage.resx#L146

Reply:
Hmmm when I try that it displays the entire string in the UI:
![image](https://github.com/dotnet/roslyn/assets/62502207/709d77a6-e094-449c-9472-007ba808eb88)

Reply:
Never mind I had misunderstood how these keywords were registered. I'm leaving both "Display inline hints" and "Display inlay hints" as search strings.